src/advect1d_vertical.jl

```diff
@@ -0,0 +1,253 @@
+#### This Julia file contains functions to implement 
```

Comment:
Putting the documentation at the top of the file is an older style; the newer way is to put the documentation (in the case of Julia, as text strings rather than comments) next to the definition of each function so that it can be used by online documentation systems (like this: https://gaschem.earthsci.dev/stable/api/)